python/src/aiconfig/editor/client/src/Editor.tsx

```diff
@@ -68,21 +68,22 @@ export default function Editor() {
     });
   }, []);
 
-  const runPrompt = useCallback(async (promptName: string) => {
-    return await ufetch.post(ROUTE_TABLE.RUN_PROMPT, {
-      prompt_name: promptName,
-    });
-  }, []);
-
-  const runPromptStream = useCallback(
-    async (promptName: string, onStream: RunPromptStreamCallback) => {
+  const runPrompt = useCallback(
+    async (
+      promptName: string,
+      enableStreaming: boolean = true,
```

Comment:
cc @rholinshead I can remove default value if you prefer

Reply:
Let's keep it. I think we should default to true so that we 'stream' by default for everything and change to regular request/response only if the model settings specifically states not to (stream: false). External implementations can default to false here if they want. We probably need to swap `onStream` to be before `enableStreaming` and make enableStreaming optional